release-notes/release-notes52.md

```diff
@@ -83,6 +84,26 @@ A new `link_default_protocol` option has been added to the Autolink and Link plu
 - For information on using this setting with the Autolink plugin, see: [Autolink plugin - `link_default_protocol`]({{site.baseurl}}/plugins/autolink/#link_default_protocol).
 - For information on using this setting with the Link plugin, see: [Link plugin - `link_default_protocol`]({{site.baseurl}}/plugins/link/#link_default_protocol).
 
+## Minor changes for TinyMCE 5.2
```

Comment:
The following are really security fixes, so not sure if you want to remove them and just put something more generic?
```
- Changes the default schema to disallow `onchange` for select elements.
- Changes the editor parser to treat CDATA nodes as bogus HTML comments to match the HTML parsing specification. A new `preserve_cdata` setting has been added to preserve CDATA nodes if required
```
Alternatively, maybe we could just add another section saying some security issues were fixed?

Reply:
changed the description a little to highlight that they are security related. Unwilling to pull them out into their own section at this time.